api-service/controller/databases.go

```diff
@@ -100,6 +100,23 @@ func (ctrl *APIController) GetUsedLicensesPerDatabasesByHost(w http.ResponseWrit
 	ctrl.GetUsedLicensesPerDatabasesJSON(w, r, hostname, *filter)
 }
 
+func (ctrl *APIController) GeGrantDbaByHost(w http.ResponseWriter, r *http.Request) {
```

Comment:
Could you rename **GeGrantDbaByHost** in **GetGrantDbaByHost?**

Reply:
I removed the function because it is no longer needed